.config/cucumber.yml

```diff
@@ -1,7 +1,7 @@
 # config/cucumber.yml
 ##YAML Template
 ---
-default: --profile html
+default: --profile html --tags ~@ci
```

Comment:
+1, except I'd call _ci_ either _ci-only_ or maybe we should even call it _long_running_. The latter makes also clear why we don't want to run this per default.

I'd also add a _all_ profile, so that we can run easily  on the dev machine all tests if we want to